gpu/octree/src/utils/morton.hpp

```diff
@@ -133,7 +133,7 @@ namespace pcl
             }	
              __device__ __host__ __forceinline__ Morton::code_t operator()(const float4& p) const
             {			
-                return (*this)(make_float3(p.x, p.y, p.z));                
+                return *this(make_float3(p.x, p.y, p.z));
```

Comment:
This change seems to break the Ubuntu 20.04 check:
`/__w/1/s/gpu/octree/src/utils/morton.hpp(136): error: expression preceding parentheses of apparent call must have (pointer-to-) function type`
Please change it back, same for bfrs.cu